e2e/run.go

```diff
@@ -31,7 +32,7 @@ var args = struct {
 	setup:    flag.String("setup", "", "Only run setups matching this regexp"),
 	run:      flag.String("run", "", "Only run tests matching this regexp"),
 	debug:    flag.Bool("debug", false, "Enables debug logs and preserves containers after running, this will run only the first matching setup"),
-	parallel: flag.Int("parallel", 1, "How many setups to run in parallel"),
+	parallel: flag.Int("parallel", runtime.NumCPU()/2+1, "How many setups to run in parallel"),
```

Comment:
Unfortunately, my Podman machine does not have access to all host's cpus. The parallelism becomes too high and some tests still fail.

Reply:
It's still DNS lookup error for services other than 'proxy'  🤔